narwhal/network/src/failpoints.rs

```diff
@@ -24,11 +24,7 @@ pub struct FailpointsResponseHandler {}
 
 impl ResponseHandler for FailpointsResponseHandler {
     fn on_response(self, _response: &anemo::Response<bytes::Bytes>) {
-        // TODO: Use tokio::sleep() instead of built in sleep()/delay()
-        // Warning: if this failpoint is used with the default sleep()
-        // or delay() it could end up blocking the system and causing other
-        // unintended effects.
```

Comment:
Is this comment no longer applicable? It seems important

Reply:
It is no longer applicable